linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_checking_systemd_timer/oval/sle15.xml

```diff
@@ -0,0 +1,52 @@
+<def-group>
```

Comment:
I suggest renaming the file to shared.xml. It would make the rule adoption easier for other products if necessary in the future.